Apps/Common/src/Common.csproj

```diff
@@ -3,8 +3,6 @@
   <PropertyGroup>    
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <RuntimeFrameworkVersion>2.2</RuntimeFrameworkVersion>
-    <!-- SonarQube needs this -->
-    <ProjectGuid>{8660083F-F582-47C6-BC98-6DCF0F4AC851}</ProjectGuid>
```

Comment:
Do you know if sonarqube will work without this?

Reply:
Yes, you can see the links in this PR, the Immunization was broken and that failed sonar qube, after I fixed it its fine!